plugins/pdr_deterministic_plugin/ufm_sim_web_service/telemetry_collector.py

```diff
@@ -23,34 +23,48 @@ def get_telemetry(self):
         get the telemetry from secondary telemetry, if it in test mode it get from the simulation
         return DataFrame of the telemetry
         """
+        # get telemetry data
         if self.test_mode:
             url = "http://127.0.0.1:9090/csv/xcset/simulated_telemetry"
         else:
             url = f"http://127.0.0.1:{self.SECONDARY_TELEMETRY_PORT}/csv/xcset/{self.SECONDARY_INSTANCE}"
         try:
-            self.logger.info(f"collecting telemetry from {url}.")
+            self.logger.info(f"Collecting telemetry from {url}.")
             telemetry_data = pd.read_csv(url)
         except (pd.errors.ParserError, pd.errors.EmptyDataError, urllib.error.URLError) as connection_error:
-            self.logger.error("failed to get telemetry data from UFM, fetched url=%s. Error: %s",url,connection_error)
+            self.logger.error("Failed to get telemetry data from UFM, fetched url=%s. Error: %s",url,connection_error)
             telemetry_data = None
-        if self.previous_telemetry_data is not None and telemetry_data is not None:
-            delta = self._get_delta(self.previous_telemetry_data,telemetry_data)
-            # when we want to keep only delta
-            if len(delta) > 0:
-                self.data_store.save(delta,self.data_store.get_filename_delta())
-        elif telemetry_data is not None:
-            # when we want to keep the abs
+
+        # store telemetry data
```

Comment:
I think moving the code to a method that saves the data to a different function will make the code clearer and clearner.

Reply:
Done